config_db_migrate/versions/bb5278995f41_run_limitation_for_products.py

```diff
@@ -0,0 +1,28 @@
+"""Run limitation for products
+
+Revision ID: bb5278995f41
+Revises: 150800b30447
+Create Date: 2018-03-01 15:38:41.164141
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'bb5278995f41'
+down_revision = '150800b30447'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Once the autogenerated operations had been revised to work, these autogenerated comments should be removed.